README.md

```diff
@@ -1 +1,46 @@
-# bff-samples
\ No newline at end of file
+## Ballerina Persists module with databases
```

Comment:
Also can we have more meaningful/understandable title for the new readers here mentioning what we are doing. Have the same title as the sample title.

Reply:
I have few suggestions
> `working_with_databases`
> `data_access_with_bal_persist`
> `bal_persist_for_data_access`

wdyt?

Reply:
How about something like below, without directly using feature/package name (as it is a new term for readers)
`Persistently-store-data-using-Ballerina`

Also use hyphes instead of underscores for sample names as it looks good in URLs.

Reply:
ok, Thank you, will update

Reply:
updated